source/settings/telemetry.txt

```diff
@@ -0,0 +1,134 @@
+.. _compass-telemetry:
+
+===========================
+Configure Telemetry Options
+===========================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+|compass-short| collects anonymous aggregated usage data to improve
+MongoDB products. |compass-short| collects this information by default,
+but you can disable this data collection.
+
+About this Task
+---------------
+
+|compass-short| telemetry tracks the following data:
+
+- The type of MongoDB deployment that you connect |compass-short| to.
+  For example, Enterprise Edition, Community Edition, or Atlas Data
+  Lake.
+
+- Actions that you perform in |compass-short|, such as creating
+  collections and indexes. Telemetry only tracks the general action
+  performed, and does not track field values or namespaces.
+
+- Errors.
+
+|compass-short| *does not* track:
+
+- IP addresses, hostnames, usernames, or credentials
+
+- Queries run in |compass-short|
```

Comment:
We will soon start tracking queries generated from the GenAI feature in an anonymized way. Maybe we can reword this so it says we won't be storing queries in a way that can be associated with the user?